setup.py

```diff
@@ -50,27 +50,27 @@
 
 
 REQUIREMENTS = [
-    'google-cloud-bigquery >= 0.20.0',
-    'google-cloud-bigtable >= 0.20.0',
-    'google-cloud-happybase >= 0.20.0',
-    'google-cloud-core >= 0.20.0',
-    'google-cloud-datastore >= 0.20.0',
-    'google-cloud-dns >= 0.20.0',
-    'google-cloud-error-reporting >= 0.20.0',
-    'google-cloud-language >= 0.20.0',
-    'google-cloud-logging >= 0.20.0',
-    'google-cloud-monitoring >= 0.20.0',
-    'google-cloud-pubsub >= 0.20.0',
-    'google-cloud-resource-manager >= 0.20.0',
-    'google-cloud-storage >= 0.20.0',
-    'google-cloud-translate >= 0.20.0',
-    'google-cloud-vision >= 0.20.0',
-    'google-cloud-runtimeconfig >= 0.20.0',
+    'google-cloud-bigquery >= 0.21.0',
```

Comment:
1. They all need to be incremented because they depend on core.
1. They are all at `0.20.0` right now because we elected to go with the "next" version after `gcloud` when upgrading.
1. Only one package (`datastore`) has had a release (`0.20.1`) since the rename